nginx.tmpl

```diff
@@ -66,11 +66,12 @@ http {
         return 200;
     }
 
-    location / {
+    location ~* ^/([^/]*) {
```

Comment:
Aaah, ok skjønner. Har ingen bedre forslag egentlig, så om det fungerer for taksonomi-tjenesten og våre APIer må det jo være greit :)